automatic/sourcetree/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,28 @@
+$ErrorActionPreference = 'Stop';
+
+$packageName = 'sourcetree'
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
```

Comment:
Could you move [this](https://github.com/pascalberger/chocolatey-coreteampackages/blob/e693ec0c26b8ac22901b54a022923fc0f8a3f859/automatic/sourcetree/tools/chocolateyInstall.ps1#L3-L8) section into the HashTable with args and change updater so it works ? Its way more DRY that way.